inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/IndexingConstants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.search.index.mtas;
+
+public class IndexingConstants {
+    public static String KB = "kb";
```

Comment:
Constants are `static final` and use only CAPITAL letters.

Reply:
DONE